zarr/util.py

```diff
@@ -19,10 +19,24 @@
 }
 
 
+# from
+# https://stackoverflow.com/questions/50916422/python-typeerror-object-of-type-int64-is-not-json-serializable/50916741
```

Comment:
I'm not sure we can take code from StackOverflow due to [licensing]( https://stackoverflow.com/help/licensing ), which would be incompatible with MIT (Zarr's licensing). Is there a way to write this where it doesn't depend on StackOverflow code?

Reply:
It's just a class. Maybe renaming it and removing all references?

The reference is just to give credit where it is due.